interfaces/PA-App/src/app/services/pa-account-api.service.ts

```diff
@@ -65,6 +65,9 @@ export class PaAccountApiService {
         false,
       )
       .pipe(
+        map((value) => {
+          return value.data ? JSON.parse(value.data) : value;
```

Comment:
The logic of parsing the returned string is moved into the API-handler.
As the API returns the `{ "data": "..." }`-style object, this handler 'should know' what to do with it...